src/components/Heading/Heading.mdx

```diff
@@ -58,17 +58,66 @@ Resources:
 - [A11y Project Accessible heading structure](https://www.a11yproject.com/posts/how-to-accessible-heading-structure/)
 - [Chakra UI Heading](https://chakra-ui.com/docs/components/typography/heading)
 
-## Default Styles
+## Updated Typographic Styles
+
+New typographic styles were introdiuced in June 2023. The new styles are the
+recommended options for the default `h` elements and the `size` overrides.
+
+## Native Heading Elements
+
+The `level` prop can be used to set a native heading element (ex. `h1`, `h2`,
+...).
+
+Based on the June 2023 updates, it is recommended to use one of the following
+values for the `level` prop:
+
+`h1`, `h2`, `h3`, `h4`, `h5`, `h6`
+
+The older `level` values and the associated styles are still available, but the
+values should be consdidered deprecated and they should only used to fulfill
+older design requirements.
+
+`one`, `two`, `three`, `four`, `five`, `size`
+
+### Recommended Options
 
 <Canvas of={HeadingStories.DefaultStyles} />
 
+### Deprecated Options
+
+<Canvas of={HeadingStories.DefaultStylesDeprecated} />
+
 ## Size Styles
 
+The `size` prop can be used to override the default styles of the native heading
+elements.
+
+Based on the June 2023 updates, it is recommended to use one of the following
+values for the `size` prop:
+
+`display1`, `heading1`, `heading2`, `heading3`, `heading4`, `heading5`,
```

Comment:
Is this meant to be something like "heading3 size is similar to an h3 size"? I'm not too big of a fan of the headingN name for size because it seems more like the level prop. But conceptually, if the first answer is yes, it makes sense.

Reply:
The default styles for `h3` are the `heading3` styles.  The idea with these two props is that you can set a level, but override the size/styles.

Do you feel the `size` values should be clearly different from from the `level` values?  Or are you saying they should be the same?

Reply:
I'm saying `heading3` is too close of a naming convention to `h3` to make it obvious that it's _only_ for size and not for the level.

Reply:
What if both props used the same values?  Would that simplify and, in a way, clarify things?  For example, what if I need a heading to be an `<h2>` element, but I need it sized as an `<h4>`.  I could set `level` as `"h2"` and set `size` as `"h4"`.

Reply:
that makes sense to me. Should we have a non-scientific pool?